packages/arb-token-bridge-ui/package.json

```diff
@@ -116,11 +118,9 @@
     "patch-package": "^8.0.0",
     "postcss": "^8.5.3",
     "postinstall-postinstall": "^2.1.0",
-    "prettier": "^2.7.1",
-    "prettier-plugin-tailwindcss": "^0.1.11",
+    "prettier": "^3.5.3",
     "satori": "^0.12.2",
     "start-server-and-test": "^2.0.11",
-    "tailwindcss": "^3.4.16",
```

Comment:
tailwindcss doesn't go under dev dependencies according to docs
this is automatically moved to deps as a result of running the migration tool
https://tailwindcss.com/docs/installation/framework-guides/nextjs